apps/prairielearn/src/ee/pages/instructorInstanceAdminLti13/instructorInstanceAdminLti13.html.ts

```diff
@@ -87,3 +333,50 @@ export function InstructorInstanceAdminLti13({
     </html>
   `.toString();
 }
+
+function lineItem(item: Lti13Lineitems) {
+  return html` <span title="${item.lineitem_id}">${item.lineitem.label}</span>`;
+  /*
+  html`
+    <!--
+    <button
+      class="btn btn-xs"
+      onClick="event.preventDefault();$(this).next('.lineitem-detail').toggle();"
+    >
+      ...
+    </button>
+    <div class="lineitem-detail" style="display:none;">
+      <pre>${JSON.stringify(item, null, 2)}</pre>
+    </div>
+  `;
+  */
+}
+
+export function LineitemsInputs(lineitems: Lti13Lineitem[]) {
+  if (lineitems.length === 0) {
+    return html`<p>None found.</p>`.toString();
+  }
+  // TODO: Filter out assignments that are already linked? Re-linking will override the
+  // previous assignment link
```

Comment:
Do you plan to address this before this PR lands?

Reply:
Probably not. Enhancing the displayed list of learning outcomes to link can come later.